coin_prices_from_coingecko/output.go

```diff
@@ -39,6 +27,14 @@ func WriteOutput(output any) uint64 {
 }
 
 func WriteError(err error) uint64 {
-	data := Result{}.JSONMarshalWithError(err)
+	result := Result{
+		Success: false,
+		Error:   err.Error(),
+		Value:   nil,
+	}
+	data, marshalErr := json.Marshal(result)
+	if marshalErr != nil {
+		basm.Log(fmt.Sprintf("Error marshalling Result: %v", marshalErr))
```

Comment:
Do we want to panic here? Right now we just log that we are unable to marshal the result and then return a ptr data (which is undefined at this point)

Reply:
If we failed to marshal then explicitly build the result string with a hardcoded error message saying something like "failed to marshal result"